lib/rules/quotes.js

```diff
@@ -28,6 +40,7 @@ export default class Quotes extends Rule {
       [
         '  * "double" - requires the use of double quotes wherever possible',
         '  * "single" - requires the use of single quotes wherever possible',
+        '  * { hbs: `single`|`double`, html: `single`|`double` } - requires different quotes for Handlebars and HTML syntax',
```

Comment:
We'll need to mention the new options in the rule documentation file.

Reply:
Looks like an update to the rule doc is still needed. Let's mention the behavior of what happens when one of these is not provided. Let's also provide an example showing what code `hbs` applies to and what code `html` applies to.